### PR TITLE
String to int bug cleared

### PR DESCRIPTION
**Pull request by mstuteja**
"Display temperature as whole number instead of decimal number #2" ***Cleared**

### CODE REVIEW

File: src/com/tt/APIUtils.java [hunk 1]
```diff
@@ -49,12 +49,12 @@
...
-//        JSONObject json = new JSONObject(str);
-//
-//        System.out.println(Arrays.toString(getAnyKeyValueAsString(json, new String[]{"weather.0.icon"})));
-//    }
+   public static void main(String[] args) {
+       String str = "{\"test\":\"ing\",\"weather\":[{\"id\":711,\"main\":\"Smoke\",\"description\":\"smoke\",\"icon\":\"50n\"}]}";
+       JSONObject json = new JSONObject(str);
+
+       System.out.println(Arrays.toString(getAnyKeyValueAsString(json, new String[]{"weather.0.icon"})));
+   }
```
Comment (PankajRPandey) [inline src/com/tt/APIUtils.java]: This was meant only for debugging purpose. You can comment or remove the main method from APIUtils class. Actually there should be junit tests for this class but we'll add them later.

File: src/com/tt/TrayUI.java [hunk 2]
```diff
@@ -93,7 +93,7 @@ public void mouseClicked(MouseEvent e) {
...
 
                         mainPanel.add(pageStartLabel, BorderLayout.PAGE_START);
-                        JLabel pageEndLabel = new JLabel("<html><center><span style='color:white;font-size:15px;'><i>Feels like " + weatherAPIKeyValue[1] + "°C</i></span><p style='color:white;font-size:12px;'>High " + weatherAPIKeyValue[2] + "°C &nbsp;&nbsp; | &nbsp;&nbsp; Low " + weatherAPIKeyValue[3] + "°C</p></center></html>", SwingConstants.CENTER);
+                        JLabel pageEndLabel = new JLabel("<html><center><span style='color:white;font-size:15px;'><i>Feels like " + Integer.parseInt(weatherAPIKeyValue[1]) + "°C</i></span><p style='color:white;font-size:12px;'>High " + weatherAPIKeyValue[2] + "°C &nbsp;&nbsp; | &nbsp;&nbsp; Low " + weatherAPIKeyValue[3] + "°C</p></center></html>", SwingConstants.CENTER);
```
Comment (PankajRPandey) [inline src/com/tt/TrayUI.java]: I apologize for not adding details. I think it's good to have ceiling (Parsing as float and then using ceiling method) values of all the temperature numbers that we get from the API. These consists of weatherAPIKeyValue[0] [1][2]&[3].

**Review by PankajRPandey — CHANGES_REQUESTED** [review]
Thanks for your time and interest on this project. There are some changes required. Please read the comments and get back to me if you have doubt.

File: src/com/tt/TrayUI.java (hunk 2 above)
Comment (mstuteja) [reply src/com/tt/TrayUI.java]: Alright. I did the changes. Please review.

File: src/com/tt/TrayUI.java [hunk 3]
```diff
@@ -85,15 +85,15 @@ public void mouseClicked(MouseEvent e) {
                             malformedURLException.printStackTrace();
                         }
 
-                        JLabel centerLabel = new JLabel("<html><center><span style='color:white;font-size:55px;'>" + weatherAPIKeyValue[0] + "°C" + "</span></center></html>", i, SwingConstants.LEFT);
+                        JLabel centerLabel = new JLabel("<html><center><span style='color:white;font-size:55px;'>" + Integer.parseInt(weatherAPIKeyValue[0]) + "°C" + "</span></center></html>", i, SwingConstants.LEFT);
```
Comment (PankajRPandey) [inline src/com/tt/TrayUI.java:88]: The numbers that we get from the API are floating point numbers so if you try parsing these number as Integer you'll get NumberFormatException. I would suggest you to parse these numbers as Float using Float.parseFloat() method and enclose this statement inside Math.ceil() function which will give you an upperbound Integer.

**Review by PankajRPandey — CHANGES_REQUESTED** [review]
Please make the necessary changes

**Pull request closed without merging**